source/agora/test/Base.d

```diff
@@ -521,6 +521,11 @@ public struct NodePair
     /// used in the Nomination protocol.
     private shared(TimePoint)* cur_time;
 
+    ///
+    public TestAPI api;
+
+    alias api this;
+
```

Comment:
Client already implicitly converts to `TestAPI`.

Reply:
>  But actually, it shouldn't need special handling if we keep a reference to the interface, does it ?

Wasn't this what you meant here?